### PR TITLE
Update About Page

### PR DESCRIPTION
**Pull request by j-alicia-long**
# Status
Done

# What I did 
## Jules
- Update team roster
- Removed apply to TCF and write a review buttons
- Removed Color blocks in the history tab
- Added sponsors tab and handshake logo
- Used apply to TCF logo for current team logo

## Jennifer
- Removed old custom js/jquery and css for navs
- Removed extraneous About views
- Updated about nav styling

## Issues addressed
#198 (Update About Page)

## Screenshots
image

### CODE REVIEW

File: tcf_website/views/team_alums.json [hunk 1]
```diff
@@ -12,18 +12,59 @@
...
+                  "img_filename": "hpathipati.jpg",
+                  "github": ""
+                  }
+                 ]
+    },
+    {
+      "group_name": "Class of 2021",
+      "members": [
+                  {"name": "Brian Yu",
+                   "role": "19-20 President",
```
Comment (vn6) [inline tcf_website/views/team_alums.json:30]: would be more consistent with the other alum profiles to take out the "19-20"

File: tcf_website/templates/about/history.html [hunk 2]
```diff
@@ -44,17 +44,6 @@ <h4>Revival 2.0</h4>
         was re-build from scratch using the Python web-framework Django.
```
Comment (vn6) [inline tcf_website/templates/about/history.html:44]:
```suggestion
        was re-built from scratch using the Python web-framework Django.
```

Comment (julesfll) [reply tcf_website/templates/about/history.html:44]: Done

File: tcf_website/views/team_alums.json (hunk 1 above)
Comment (julesfll) [reply tcf_website/views/team_alums.json:30]: Done

File: tcf_website/templates/about/sponsors.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    <div class="sponsors-text">
+      <p>
+        Our sponsors help us cover server costs from the large amount of traffic that we get.
+        Their help has allowed the site to stay up and running continuously as we keep developing
+        new features.
+      </p>
+      <hr>
+    </div>
+    <img class="sponsors-img1 img-fluid" src="{% static 'about/history/UVA-parents-fund-logo.png' %}"/>
+    <img class="sponsors-img2 img-fluid" src="{% static 'about/history/UVA-student-council-logo.png' %}"/>
```
Comment (alex-shen1) [inline tcf_website/templates/about/sponsors.html:16]: Is the idea to make a new class for every new sponsor...? Might not be great for future maintenance, but we probably won't have that many sponsors ever so it's not a big deal.

**Review by alex-shen1 — APPROVED** [review]

File: tcf_website/templates/about/sponsors.html (hunk 3 above)
Comment (julesfll) [reply tcf_website/templates/about/sponsors.html:16]: It has to do with the grid positioning
there might be a cleaner way to do it but idk

**Pull request merged**